config.md

```diff
@@ -233,6 +234,7 @@ _Note: symbolic name for uid and gid, such as uname and gname respectively, are
     "user": {
         "uid": 1,
         "gid": 1,
+        "umask": 63,
```

Comment:
i get that they're just ints, but i wish octals were a viewable in json, and 077 wouldn't be 77